HexCtrl/src/HexUtility.h

```diff
@@ -46,8 +46,8 @@ namespace HEXCTRL::INTERNAL
 	//Multibyte to Wide string convertion.
 	[[nodiscard]] std::wstring str2wstr(std::string_view str, UINT uCodePage = CP_UTF8);
 
-	//Substitute all unprintable wchar symbols with dot.
-	void ReplaceUnprintable(std::wstring& wstr, bool fASCII, bool fCRLF = true);
+	//Substitute all unprintable wchar symbols with specified character
+	void ReplaceUnprintable(std::wstring& wstr, bool fASCII, wchar_t wcReplacementChar = L'.', bool fCRLF = true);

```

Comment:
We don't need default value if we gonna always call a method with a provided arg (m_wcUnprintable in this case).
`fCRLF` default could also be omitted here.

Reply:
I added the new argument before the existing optional fCRLF argument. I would suggest either (a) This argument should not be optional and thus go before the optional fCRLF argument OR (b) If optional, it can go after the fCRLF argument. Please can you confirm which you would prefer?

Reply:
`void ReplaceUnprintable(std::wstring& wstr, bool fASCII, bool fCRLF, wchar_t wchUnprintable);`

Reply:
Changed as requested.